test/index.ts

```diff
@@ -28,7 +28,15 @@ const test = (
 	});
 };
 
-test('no types for an empty model', {}, '');
+test(
+	'no types for an empty model',
+	{},
+	`
+
+export default interface Model {
+
+}`,
```

Comment:
`balena-lint` will want this to be `export default interface Model {}`

Reply:
Although I guess we're also outputting `// Synonyms` inside the brackets so it would actually expect it to be:

```
export default interface Model {
    // Synonyms
}
```

Although like I mentioned in my other comment, we probably don't want to output this synonyms comment if no synonyms are defined.